lib/common/adjust_zone.js

```diff
@@ -1,17 +1,16 @@
 var adjust_lon = require('./adjust_lon');
 
 module.exports = function(zone, lon) {
-  if (!zone) {
+  if (zone === undefined) {
     zone = Math.floor((adjust_lon(lon) + Math.PI) * 30 / Math.PI);
 
     if (zone < 0) {
       return 0;
-    }
-    else if (zone >= 60) {
+    } else if (zone >= 60) {
       return 59;
     }
-  }
-  else {
+    return zone;
```

Comment:
I wrote about only the changes in `utm.js`, because when I changed the check in my `etmerc` branch it worked fine with the zone 1, but I see now that I did not look deep enough. So thank you for noticing this, because this could cause a separate kind of bug. Can you add a unit test for this?